Assets/Scripts/Game/Questing/QuestMCP.cs

```diff
@@ -105,6 +112,16 @@ public override string Pronoun3()
                 }
             }
 
+            public override string VampireNpcClan()
+            {   // %vcn
+                if (parent.LastResourceReferenced == null && !(parent.LastResourceReferenced is Person))
+                    return null;
+
+                // TODO: How to find vampire clan of an NPC? Is faction data as used here the right method that covers all cases?
```

Comment:
I'm certain you can just use the region NPC is in to determine their vampire clan name. The following should do it:

    VampireClans vampireClan = FormulaHelper.GetVampireClan(regionIndex);
    string clanName = TextManager.Instance.GetText("Races", vampireClan.ToString().ToLower());